frontend/pages/events/[id]/about.vue

```diff
@@ -37,13 +37,14 @@
     </HeaderAppPage>
     <div class="pb-6 space-y-6">
       <div
-        class="lg:grid space-y-6 lg:grid-cols-3 lg:grid-rows-1 lg:space-y-0"
```

Comment:
We can also remove `lg:space-y-0` from this line here, @hoang227 :) All good though! Let me send this along and I'll merge it in :)